main.cpp

```diff
@@ -116,7 +109,33 @@ Midi Controller
     3) change bank
  */
 
+/*
+Thing 10) House
+5 properties:
+    1) Number of Doors
+    2) Type of Roof
+    3) Color of Paint
+    4) Size of Windows
+    5) Acreage
+3 things it can do:
+    1) House people
+    2) Store Furniture
+    3) Protect people
+ */
```

Comment:
You are going to have a very hard time coming up with 5 properties and 3 actions for each of those 5 properties in your House object
- [ ] come up with a better object for `#10` that can be easily split into sub-objects.